### PR TITLE
Closeall args

### PR DESCRIPTION
**Pull request by OliverCordingl1**
#### Information

> Check one

- [ ] This includes major changes (breaking changes)
- [x] This includes minor changes (minimal usage changes, minor new features)
- [ ] This includes patches (bug fixes, documentation changes etc)

#### Is this related to an issue?

Finished #85

#### Changes made

Adds arguments to close all tickets older than a specific date. Also modifies the database entry for each message in a ticket channel so `-closeall <timestamp>` closes from the last message, not creation date.

#### Confirmations

> Check all that apply

- [ ] I have updated any necessary documentation
- [x] This uses consistent code style
- [x] This is tested and works

### CODE REVIEW

File: src/commands/closeall.js [hunk 1]
```diff
@@ -12,15 +12,18 @@ const fs = require('fs');
...
+const toTime = require('to-time-monthsfork');
 
 // A slight modification to the 'close' command to allow multiple tickets to be closed at once
 
 module.exports = {
 	name: 'closeall',
-	description: 'Closes all currently open tickets.',
-	usage: '',
+	description: 'Closes all currently open tickets older than a specified time length',
+	usage: '<timestamp>',
```
Comment (eartharoid) [inline src/commands/closeall.js:24]: Should be `[time]`

File: src/commands/closeall.js [hunk 2]
```diff
@@ -12,15 +12,18 @@ const fs = require('fs');
...
 	aliases: ['ca'],
-	example: 'closeall',
+	example: 'closeall 1y 2m',
```
Comment (eartharoid) [inline src/commands/closeall.js:26]: Should be `closeall 1y 2mo`, although I think a shorter period of `1mo` or `1mo 1w` would be better.

**Review by eartharoid — COMMENTED** [review]
Looks ok, but there's a problem with your to-time fork: [TimeFrame.js#L31](https://github.com/OliverCordingl1/to-time/blob/master/src/TimeFrame.js#L31), it says `this.year` instead of `this.month`.

**Comment by OliverCordingl1** [comment]
Fixed it, should be fine now. Anything else you want me to sort?

Also fixed the error you pointed out in to-time fork. Thanks.

**Pull request merged**